crates/isograph_compiler/src/create_schema.rs

```diff
@@ -86,10 +86,30 @@ pub fn create_schema<TNetworkProtocol: NetworkProtocol + 'static>(
 
     for (parent_object_entity_name, expose_as_fields_to_insert) in expose_as_field_queue {
         for expose_as_field in expose_as_fields_to_insert {
-            let unprocessed_scalar_item = unvalidated_isograph_schema
-                .create_new_exposed_field(expose_as_field, parent_object_entity_name)?;
-
-            unprocessed_items.push(SelectionType::Scalar(unprocessed_scalar_item));
+            match unvalidated_isograph_schema
+                .create_new_exposed_field(expose_as_field, parent_object_entity_name)
```

Comment:
let's have `create_new_exposed_field` return a `Result<Option<...>>` and leave this code unchanged